gradle.properties

```diff
@@ -12,3 +12,12 @@ enableCrossCompileToWindowsAarch64=false
 enableCrossCompileToWindowsX86_64=false
 
 kdt.win32.angle-version=ec4d8f8e4d
+
+# Absolute path to the `cargo` executable if it's not in the PATH used by both terminal and IDE(s)
+rust.cargoCommand=
+
+# Absolute path to the `rustup` executable if it's not in the PATH used by both terminal and IDE(s)
+rust.rustupCommand=
+
+# Absolute path to the `rustc` executable if it's not in the PATH used by both terminal and IDE(s)
+rust.rustcCommand=
```

Comment:
We use rustc only to check the version. Can't we left only `cargo` command here?